Updates for W3S at scale and adding architecture (that page needs work and diagram). (#254)
Consensys/doc.web3signer: docs/how-to/store-keys/index.md

| [YubiHSM 2]                          |     x     |  x  |
| [USB Armory Mk II]                   |     x     |  x  |

> Web3Signer supports `Eth1` signing from HSMs and Vaults, but needs to load private keys into memory for `Eth2` signing.  Make sure to follow [best practices](../../get-started/key-best-practices.md) when storing private keys.

Reviewer: ```suggestion
Web3Signer supports `Eth1` signing from HSMs and vaults, but must load private keys into memory for `Eth2` signing. Follow [best practices](../../get-started/key-best-practices.md) when storing private keys.
```
